src/contollers/block.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  Route,
+  Get,
+  Post,
+  Body,
+  Query,
+  Tags,
+  TsoaResponse,
+  Response,
+  Res,
+} from "tsoa";
+import { ifClient } from "../utils/ironfish"; // Adjust the import path as needed
+import { lightBlockCache } from "../cache"; // Adjust the import path as needed
+import { LightBlock } from "../models/lightstreamer";
+
+@Route("/")
+@Tags("Block Controller")
+export class BlockController {
+  @Get("latest-block")
+  public async getLatestBlock() {
+    const rpcClient = await ifClient.getClient();
+    const response = await rpcClient.chain.getChainInfo();
+    return {
+      sequence: Number(response.content.currentBlockIdentifier.index),
+      hash: response.content.currentBlockIdentifier.hash,
+    };
+  }
+
+  @Post("transaction")
+  @Response<Error>("400", "Error broadcasting transaction")
+  public async postTransaction(
```

Comment:
I think it'd be good to change the name from `postTransaction` since that implies creating proofs + signing a transaction. I think the name might be autogenerated by the swagger generator? I noticed it when looking at the examples, `api.transaction.postTransaction(`

Maybe `broadcastTransaction` to match the RPC endpoint?